### PR TITLE
Support unaligned resources

### LINKED ISSUE #38: Resourcelib fails to parse Jetbrains' executables resources
Feeding the (64-bit) executables from [Jetbrains'](https://www.jetbrains.com) tools (eg. [[1](https://www.jetbrains.com/webstorm/download/#section=windows-version)], [[2](https://www.jetbrains.com/datagrip/download/#section=windows-version)]) to ResourceLib fails with an exception:

```
Vestris.ResourceLib.LoadException: Error loading 'datagrip64.exe'. Unknown error (0xe0434352).
---> System.Exception: Error loading resource '1' RT_VERSION (1031).
---> System.ArgumentException: Item has already been added. Key in dictionary: '\u0001StringFileInfo'  Key being added: '\u0001StringFileInfo'
   at Vestris.ResourceLib.ResourceInfo.EnumResourceLanguages(IntPtr hModule, IntPtr lpszType, IntPtr lpszName, UInt16 wIDLanguage, IntPtr lParam)
--- End of inner exception stack trace ---
--- End of inner exception stack trace ---
   at Vestris.ResourceLib.ResourceInfo.Load(String filename)
```

### PR DESCRIPTION
**Pull request by hakito**
Instead of trying to parse the unallocated resource buffer, an allocated copy is used for processing.

After implementing the fix the unit test _TestReadWriteResourceBytes_ failed for a FontDirectoryResource. The problem was that FontDirectoryResource was copying one byte **after** the actual end of the Resource to the _reserved array.

It seems it was copying the byte because the FontDirectoryWrite entry did not write null terminated strings.

This fixes #38

### CODE REVIEW

**Comment by hakito** [comment]
Unfortonately I cannot provide the assembly. But in pull request https://github.com/resourcelib/resourcelib/pull/39 is an assembly that has the same problem. I merged with the other fork.

**Comment by dblock** [comment]
Looks great, thank you! Care to update https://github.com/resourcelib/resourcelib/blob/master/CHANGELOG.md please and I'll merge?

**Comment by dblock** [comment]
I also could use some co-maintainer help, you seem to know what you're doing @hakito, interested?

**Comment by hakito** [comment]
I updated the changelog and did also some polishing in Aligned.cs. (Yesterday I was in a hurry.)

Regarding co-maintainer:

I cannot guarantee to have time for it, but I can try.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,10 +4,12 @@
 
 * [#59](https://github.com/resourcelib/resourcelib/pull/59): Update to Visual Studio 2017; resourcelib now targets .NET Frameworks 2.0, 3.5, 4.0, and 4.5, and .NET Standard 2.0 - [@thoemmi](https://github.com/thoemmi).
 * [#62](https://github.com/resourcelib/resourcelib/issues/62): ResourceLib 2.0 beta is no longer strong-named.
+* [#66](https://github.com/resourcelib/resourcelib/pull/66): Added support for unaligned resources.
```
Comment (dblock) [inline CHANGELOG.md]: Add you name in the end, like ` - [@hakito](https://github.com/hakito).` Also maybe fix the one above from #62.

**Comment by dblock** [comment]
I merged this.

**Comment by dblock** [comment]
I [invited you](https://github.com/resourcelib/resourcelib/invitations) @hakito to collaborators. Thanks for helping out!

**Comment by thoemmi** [comment]
Thanks for your contribution @hakito. Thanks to CI, your PR is included in [2.0.0-beta0032](https://www.nuget.org/packages/Vestris.ResourceLib/2.0.0-beta0032).

**Pull request merged**